src/integrationTest/java/uk/gov/hmcts/darts/arm/service/ArchiveRecordServiceIntTest.java

```diff
@@ -185,7 +185,7 @@ void generateArchiveRecord_WithNleMediaProperties_ReturnFileSuccess() throws IOE
         dartsDatabase.getExternalObjectDirectoryRepository().saveAndFlush(armEod);
 
         when(armDataManagementConfiguration.getMediaRecordClass()).thenReturn(DARTS);
-        when(armDataManagementConfiguration.getMediaRecordPropertiesFile()).thenReturn("tests/arm/properties/nle/media-record.properties");
+        when(armDataManagementConfiguration.getMediaRecordPropertiesFile()).thenReturn("tests/arm/properties/media-record.properties");
```

Comment:
The methods with Nle in name will no longer be required as they be the same as live. Since Live is being removed you may as remove that as well